packages/protocol/migrations/16_elect_validators.ts

```diff
@@ -18,10 +23,29 @@ const argv = minimist(process.argv, {
   default: { keys: '' },
 })
 
+const gethRepoPath = argv.localgeth || '/tmp/geth'
+
 function serializeKeystore(keystore: any) {
   return Buffer.from(JSON.stringify(keystore)).toString('base64')
 }
 
+export function execCmdWithOutput(cmd: string, args: string[], options: any = {}): Promise<Buffer> {
```

Comment:
I think we already have a similar helper defined elsewhere in the monorepo, `execCmd`. We haven't been super clean about it, I think it's copied-pasted in several spots. Might want to consider using the same implementation just to stay consistent, or even importing it from `@celo/celotool/src/lib/utils`.

Reply:
Should have been gone.